diwako_cbrn.vr/scripts/cbrn/functions/fn_detectorGeigerPFH.sqf

```diff
@@ -0,0 +1,45 @@
+_low_rad = ["cbrn_low_rad_0", "cbrn_low_rad_1", "cbrn_low_rad_2", "cbrn_low_rad_3"];
+_mid_rad = ["cbrn_mid_rad_0", "cbrn_mid_rad_1", "cbrn_mid_rad_2", "cbrn_mid_rad_3"];
+_high_rad = ["cbrn_high_rad_0", "cbrn_high_rad_1", "cbrn_high_rad_2", "cbrn_high_rad_3"];
+_danger_rad = ["cbrn_danger_rad_0", "cbrn_danger_rad_1", "cbrn_danger_rad_2", "cbrn_danger_rad_3"];
+_deadly_rad = ["cbrn_deadly_rad_0", "cbrn_deadly_rad_1", "cbrn_deadly_rad_2", "cbrn_deadly_rad_3"];
+_run_rad = ["cbrn_run_rad_0", "cbrn_run_rad_1", "cbrn_run_rad_2", "cbrn_run_rad_3"];
+_omg_rad = ["cbrn_omg_rad_0", "cbrn_omg_rad_1", "cbrn_omg_rad_2", "cbrn_omg_rad_3"];
+
+if (!isGameFocused || isGamePaused) exitWith {};
+params ["_args", "_idPFH"];
+_args params ["_lastIteration"];
+
+private _time = cba_missiontime;
+private _delta = _time - _lastIteration;
+_args set [0, _time];
+
+if ( !cbrn_geiger ||  {!(ace_player getVariable ["cbrn_detector_geiger", false])} ) exitWith {
+    cbrn_geigerPfh = -1;
+    [_idPFH] call CBA_fnc_removePerFrameHandler;
+};
+
+if (cbrn_curThreat < 0.25) exitWith {};
+
+if (cbrn_lastGeiger <= _time) then {
```

Comment:
Hm. instead of checking each 0.05 second if 0.5 seconds have passed, why not just make the pfh run at 0.5?

Reply:
I can forsee some issues with the PFH running that slow in some very extreme edge cases (going into a Lvl3 radius just after the PFH should've fired for example), especially considering the decision of making the sound bites longer, but I do agree that it running this often should be changed. Does 0.33 sound okay to you?